docs/integrations/ceramicExample.md

```diff
@@ -16,21 +16,23 @@ Learn how to use Lit Protocol to encrypt messages to yourself and save them to t
 
 The Ceramic Protocol is built on decentralized event streams, where user accounts (enabled by decentralized identifiers, or [DIDs](https://developers.ceramic.network/protocol/accounts/decentralized-identifiers/)) cryptographically sign data events and submit them to the network. These events are stored in the Interplanetary File System (IPFS) using the IPLD protocol, and organized into readable streams. Each stream is flexible enough to store many types of content. Therefore, Ceramic is home to a diversity of different data use cases such as user profiles, posts, relations to other entities, and more.
 
-Due to Ceramic's open readability, any participating node can read from any stream in the Network. Therefore, encrypting data using Lit Protocol and saving it Ceramic is a common (and necessary) integration across many teams.
+Due to Ceramic's open readability, any participating node can read from any stream in the network. Therefore, encrypting data using Lit Protocol and saving it Ceramic is a common (and necessary) integration for many teams.
 
 ### Ceramic as a Data Ledger
 
 It's important to note that Ceramic can be viewed as a "Data Ledger" middle ground between on and off-chain activity. All Ceramic events are periodically rolled into a Merkle tree and the root is published to the Ethereum blockchain, thus ensuring consensus on the global transactions ordering within Ceramic.
 
 For more information on how Ceramic works, visit [How it Works](https://ceramic.network/how-it-works).
 
-### ComposeDB
+## ComposeDB
 
 ComposeDB is a graph database built on Ceramic that offers further composability and ease of querying to developers. ComposeDB delivers native support for GraphQL and also splits read/write load by default for additional performance. When running a Ceramic node with ComposeDB, developers can define their own [data models](https://composedb.js.org/docs/0.5.x/create-your-composite) using GraphQL, or choose to begin indexing on existing data models already defined by the community, or both! 
 
 For the purpose of this tutorial, we will be using Ceramic with ComposeDB to illustrate how developers can generate, store, and query simple encrypted messages.
 
-This tutorial will use a message board example application to show how to create encrypted messages using Lit Protocol and save message instances to the Ceramic Network using ComposeDB, and will reference this [example repository](https://github.com/mzkrasner/lit-composedb).
+This tutorial will use a message board example application to show how to create encrypted messages using Lit Protocol and save message instances to the Ceramic Network using ComposeDB.
+
+To follow along, reference this [example repository](https://github.com/mzkrasner/lit-composedb).
```

Comment:
hello @debbly ! I'm glad you caught this - this should actually be https://github.com/ceramicstudio/lit-composedb (forgot to update it)

Reply:
Awesome, can you update it in the file?